Use of undeclared type or module `ErrorKind` from nom-4 macros

- Rust version: 1.26.0
- nom version: 4.0
- nom compilation features used: default

This test case:

```
#[macro_use]
extern crate nom;

use std::str;

fn atom_specials(c: u8) -> bool {
    true
}

named!(capability<&str>, do_parse!(
    tag_s!(" ") >>
    atom: map_res!(take_till1_s!(atom_specials), str::from_utf8) >>
    ("a")
));
```

results in:

```
error[E0433]: failed to resolve. Use of undeclared type or module `ErrorKind`
  --> src/main.rs:9:1
   |
9  | / named!(capability<&str>, do_parse!(
10 | |     tag_s!(" ") >>
11 | |     atom: map_res!(take_till1_s!(atom_specials), str::from_utf8) >>
12 | |     ("a")
13 | | ));
   | |___^ Use of undeclared type or module `ErrorKind`
   |
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -82,6 +82,9 @@ name = "ini_str"
 required-features = ["alloc"]
 
 [[test]]
+name = "issue_759"
+
+[[test]]
 name = "issues"
 required-features = ["alloc", "regexp_macros"]
 

--- a/src/bytes.rs
+++ b/src/bytes.rs
@@ -596,7 +596,7 @@ macro_rules! take_till (
 macro_rules! take_till1 (
   ($input:expr, $submac:ident!( $($args:tt)* )) => (
     {
-      use $crate::InputTakeAtPosition;
+      use $crate::{ErrorKind, InputTakeAtPosition};
       let input = $input;
       input.split_at_position1(|c| $submac!(c, $($args)*), ErrorKind::TakeTill1)
     }

--- a/tests/issue_759.rs
+++ b/tests/issue_759.rs
@@ -1,0 +1,18 @@
+#[macro_use]
+extern crate nom;
+
+use std::str;
+
+fn atom_specials(c: u8) -> bool {
+    c == b'q'
+}
+
+named!(
+    capability<&str>,
+    do_parse!(tag_s!(" ") >> _atom: map_res!(take_till1!(atom_specials), str::from_utf8) >> ("a"))
+);
+
+#[test]
+fn issue_759() {
+    assert_eq!(capability(b" abcqd"), Ok((&b"qd"[..], "a")));
+}

